.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file should not be part of the git repo

Reply:
As yes... the perils of using `git add --all`.